src/components/ChartRow.js

```diff
@@ -171,6 +177,24 @@ export default class ChartRow extends React.Component {
         this.setState({ yAxisScalerMap: scalerMap });
     }
 
+    componentWillMount() {
+        // Our chart scales are driven off a mapping between id of the axis
+        // and the scale that axis represents. Depending on the transition time,
+        // this scale will animate over time. The controller of this animation is
+        // the ScaleInterpolator. We create new Scale Interpolators here for each
+        // axis id.
+        this.scaleMap = {};
+        this.updateScales(this.props);
+    }
+
+    /**
+     * When we get changes to the row's props we update our map of
+     * axis scales.
+     */
+    componentWillReceiveProps(nextProps) {
```

Comment:
Isn't there already a `componentWillReceiveProps` defined above? Line 129

Reply:
See above. There's only one `componentWillReceiveProps` in the previous state of the branch.